test/Microsoft.Azure.WebJobs.Host.EndToEndTests/AzureStorageEndToEndTests.cs

```diff
@@ -74,14 +74,17 @@ public AzureStorageEndToEndTests(TestFixture fixture)
         /// - blob name pattern binding
         /// </summary>
         public static void BlobToQueue(
-            [BlobTrigger(ContainerName + @"/{name}")] CustomObject input,
+            [BlobTrigger(ContainerName + @"/{name}")] string input,
             string name,
             [Queue(TestQueueNameEtag)] out CustomObject output)
         {
+            // TODO: Use CustomObject as param when POCO blob supported
```

Comment:
Is this because we've disabled the custom CloudBlobStreamBinder stuff in this PR? I think we have a bug for that - recommend referencing that bug here so we come back and address this.

Reply:
Updated comment to link to https://github.com/Azure/azure-webjobs-sdk/issues/995.